dts/riscv/telink/telink_b91.dtsi

```diff
@@ -23,6 +23,12 @@
 			reg = <0>;
 			clock-frequency = <24000000>;
 			compatible ="telink,b91", "riscv";
+			hlic: interrupt-controller {
```

Comment:
Most platforms in Zephyr calls this either `hlic*` or in SMP configurations, it's sometimes called `CPU*_intc`. It's also referred to the `clic` described in the spec.
https://github.com/riscv/riscv-fast-interrupt/blob/master/clic.adoc

This looks OK to me though .